include/mata/nfa/nfa.hh

```diff
@@ -389,6 +394,15 @@ public:
      * @return True if some new transition (and sink state) was added to the automaton.
      */
     bool make_complete(const Alphabet& alphabet) { return this->make_complete(alphabet, this->num_of_states()); }
+
+    /**
+     * Complement deterministic NFA in-place by adding a sink state and swapping final and non-final states.
+     * @param[in] symbols Symbols needed to make the automaton complete.
+     * @param[in] sink_state State to be used as a sink state. Adds a new sink state when not specified.
+     * @return NFA complemented in-place.
```

Comment:
NFA or DFA? Or maybe just "Automaton complemented in-place".

Reply:
I will use `DFA` here to further emphasize the deterministic-ness.